flybywiresim/a32nx
# feat(mcdu): Cycle Nav DB's to reset FPlan, Weights, Fuel Data & FMS position - New PR

Fixes #6216

## Summary of Changes

Adds the ability to "recycle" Nav databases in order to clear all flight plans and fuel data. PERF data remains intact. The A32nx uses only ONE NavDB, therefore the dates for both DB's are expected to be the same in the FMGC menu.
## Screenshots (if necessary)




## References

![image](https://user-images.githubusercontent.com/42608809/163137920-adaafd29-9afb-491e-88ac-e66a01fe079b.png)





## Additional context



Discord username (if different from GitHub):Pieter#2782

## Testing instructions


- Load the sim cold & dark
- Start up and import flightplan from Simbrief, or add your own
- Go MCDU MENU button and select the FMGC system
- Note the secondary NavbDB is active ( in cyan color )
- Select the secondary Nav DB by clicking Left LSK[2]
- A momentary "PLEASE WAIT" message appears
- Flight plan should now be deleted as well as any temp flight plans that may have existed.
- PERF data ( if filled out ) should still be intact
- NOTE: Fuel data should also be removed when cycling the NAV DB, but this is not yet implemented.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
